<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_leere_unterurdner_entfernen</class>
 <widget class="QDialog" name="Dialog_leere_unterurdner_entfernen">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>602</width>
    <height>173</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>leere Unterordner entfernen</string>
  </property>
  <widget class="QPushButton" name="pushButton_start">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>581</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>581</width>
     <height>113</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="2">
     <widget class="QCheckBox" name="checkBox_cnc">
      <property name="text">
       <string>CNC-Wurzelverzeichnis</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QCheckBox" name="checkBox_soptidat">
      <property name="text">
       <string>SOptiDat</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QCheckBox" name="checkBox_anderer_pfad">
      <property name="text">
       <string>anderer Pfad:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="lineEdit_anderer_pfad"/>
    </item>
    <item row="3" column="2">
     <widget class="QPushButton" name="pushButton_anderer_pfad">
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QCheckBox" name="checkBox_an_pios">
      <property name="text">
       <string>Auftr√§ge an POIS</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
